src/styles/blocks/cards.scss

```
@@ -0,0 +1,24 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+}
+
+@media (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

You can rewrite like this
```suggestion
.cards {
  display: grid;
  grid-template-columns: repeat(1, 200px);
  row-gap: 46px;
  column-gap: 48px;
  
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }

  @media (min-width: 768px) {
    grid-template-columns: repeat(3, 200px);
  }

  @media (min-width: 1024px) {
    grid-template-columns: repeat(4, 200px);
  }
}
```